### ISSUE TITLE
`Description` field missing on Linux but present in Windows

### ISSUE THREAD

**Issue by eeintech** [openscopeproject/InteractiveHtmlBom#323]
## Windows 10

`Description` field is present:

![image](https://user-images.githubusercontent.com/4020546/189171543-50c7a1c6-df7f-4c0a-aeb9-7245dac8fe77.png)

## Linux (Ubuntu 22.04)

`Description` field is **missing**:

![image](https://user-images.githubusercontent.com/4020546/189172280-eb5636a8-8e32-4b19-89db-73b0ff029005.png)

**Comment by qu1ck** [comment]
Description field shows up only if netlist is used as extra data file. Ibom automatically picks the latest modified file as source of extra data and on your linux machine it probably happens to be the pcb file and not netlist file.

**Comment by eeintech** [comment]
Thanks, we found out what was happening:
- In Linux, you were right, your plugin did not pick explicitly the `.net` file because it was a bit older than the `.kicad_pcb` file. After pointing to the `.net` file, the `Description` field appears.
- On Windows, my colleague had generated a BOM using the `KiBOM` tool which also generates a `.xml` file from the schematics, and it appears that your plugin also picks `.xml` files as source of extra fields which is why the `Description` field was showing even though we did not explicitly export the netlist.

Mystery solved :smile: Thanks again!
